Apply gradle-java-distribution --> sls-packaging renaming

uschi2000 opened the pull request:
<!-- Reviewable:start -->
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/palantir/sls-packaging/172)
<!-- Reviewable:end -->

Commit:
Apply gradle-java-distribution --> sls-packaging renaming

markelliot commented:
you're missing the associated META-INF resource changes?

markelliot commented on build.gradle:
```diff
@@ -31,7 +31,7 @@ if (System.env.CIRCLE_TEST_REPORTS) {
 }
 
-group 'com.palantir.gradle.javadist'
+group 'com.palantir.gradle.sls-packaging'
```
note that when this changes we'll need to publish to a different bintray destination and we'll need to get that whitelisted to jcenter.

uschi2000 replied:
yep

uschi2000 commented:
Yep, aware of the bintray thing

On Mon, Jan 30, 2017, 12:20 Mark Elliot <notifications@github.com> wrote:

> *@markelliot* commented on this pull request.
> ------------------------------
>
> In build.gradle
> <https://github.com/palantir/sls-packaging/pull/172#pullrequestreview-19135333>
> :
>
> > @@ -31,7 +31,7 @@ if (System.env.CIRCLE_TEST_REPORTS) {
>      test.reports.junitXml.destination = new File(System.env.CIRCLE_TEST_REPORTS, getName())
>  }
>
> -group 'com.palantir.gradle.javadist'
> +group 'com.palantir.gradle.sls-packaging'
>
> note that when this changes we'll need to publish to a different bintray
> destination and we'll need to get that whitelisted to jcenter.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/palantir/sls-packaging/pull/172#pullrequestreview-19135333>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AGOdwX2ArK0tMOJjWLosT2kUlEjbvJZmks5rXhurgaJpZM4LxqQa>
> .
>

uschi2000 commented:
Was going to do the meta INF thing in a separate PR. Artifact names and
plugin names are unrelated.

On Mon, Jan 30, 2017, 12:49 Robert Fink <rf@robertfink.de> wrote:

> Yep, aware of the bintray thing
>
> On Mon, Jan 30, 2017, 12:20 Mark Elliot <notifications@github.com> wrote:
>
> *@markelliot* commented on this pull request.
> ------------------------------
>
> In build.gradle
> <https://github.com/palantir/sls-packaging/pull/172#pullrequestreview-19135333>
> :
>
> > @@ -31,7 +31,7 @@ if (System.env.CIRCLE_TEST_REPORTS) {
>      test.reports.junitXml.destination = new File(System.env.CIRCLE_TEST_REPORTS, getName())
>  }
>
> -group 'com.palantir.gradle.javadist'
> +group 'com.palantir.gradle.sls-packaging'
>
> note that when this changes we'll need to publish to a different bintray
> destination and we'll need to get that whitelisted to jcenter.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/palantir/sls-packaging/pull/172#pullrequestreview-19135333>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AGOdwX2ArK0tMOJjWLosT2kUlEjbvJZmks5rXhurgaJpZM4LxqQa>
> .
>
>

markelliot commented:
The META-INF thing maps plugin name to implementation. If you change the plugin name, the resource also needs to change.

Sent from my iPhone

On Jan 30, 2017, at 12:50 PM, Robert Fink <notifications@github.com<mailto:notifications@github.com>> wrote:

Was going to do the meta INF thing in a separate PR. Artifact names and
plugin names are unrelated.

On Mon, Jan 30, 2017, 12:49 Robert Fink <rf@robertfink.de<mailto:rf@robertfink.de>> wrote:

> Yep, aware of the bintray thing
>
> On Mon, Jan 30, 2017, 12:20 Mark Elliot <notifications@github.com<mailto:notifications@github.com>> wrote:
>
> *@markelliot* commented on this pull request.
> ------------------------------
>
> In build.gradle
> <https://github.com/palantir/sls-packaging/pull/172#pullrequestreview-19135333>
> :
>
> > @@ -31,7 +31,7 @@ if (System.env.CIRCLE_TEST_REPORTS) {
> test.reports.junitXml.destination = new File(System.env.CIRCLE_TEST_REPORTS, getName())
> }
>
> -group 'com.palantir.gradle.javadist'
> +group 'com.palantir.gradle.sls-packaging'
>
> note that when this changes we'll need to publish to a different bintray
> destination and we'll need to get that whitelisted to jcenter.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/palantir/sls-packaging/pull/172#pullrequestreview-19135333>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AGOdwX2ArK0tMOJjWLosT2kUlEjbvJZmks5rXhurgaJpZM4LxqQa>
> .
>
>

—
You are receiving this because you were assigned.
Reply to this email directly, view it on GitHub[github.com]<https://urldefense.proofpoint.com/v2/url?u=https-3A__github.com_palantir_sls-2Dpackaging_pull_172-23issuecomment-2D276134880&d=DwMFaQ&c=izlc9mHr637UR4lpLEZLFFS3Vn2UXBrZ4tFb6oOnmz8&r=Vp81aOxWZvuVgVK_wp-VF3pIYG92B19LcCw6XKeYC0U&m=IL8OC9q8zIMW98ClAhkwYgJ72ybZ4xRCsPtMBPRp6yQ&s=foib-bNrWoZzVQny5ZK78YLHH4aVmbZvLkm6ZISzC_k&e=>, or mute the thread[github.com]<https://urldefense.proofpoint.com/v2/url?u=https-3A__github.com_notifications_unsubscribe-2Dauth_AAp3Q9I2T8cpieSPXPB2oRkxGBQ9G9nlks5rXiK-2DgaJpZM4LxqQa&d=DwMFaQ&c=izlc9mHr637UR4lpLEZLFFS3Vn2UXBrZ4tFb6oOnmz8&r=Vp81aOxWZvuVgVK_wp-VF3pIYG92B19LcCw6XKeYC0U&m=IL8OC9q8zIMW98ClAhkwYgJ72ybZ4xRCsPtMBPRp6yQ&s=5X_Okt5UaSKwvDuqr5X2HdCCfEqrmN22BOYLo16x1dY&e=>.

uschi2000 commented:
Ah, seems like the gradle plugin publishing was borked already after we introduced the second plugin. Any way, I reverted the name change for the plugin here.

Commit:
don"t change the plugin name
